ui/app/selectors/selectors.js

```diff
@@ -475,3 +481,11 @@ export function getIsSwapsChain(state) {
   const chainId = getCurrentChainId(state);
   return ALLOWED_SWAPS_CHAIN_IDS[chainId];
 }
+
+export function getPrimaryTokenImage(state) {
+  const chainId = getCurrentChainId(state);
+
+  return chainId === BNB_CHAIN_ID
```

Comment:
I may want to add another check here for Mainnet ETH so we show the traditional light image as we did before.